trakt/tv.py

```diff
@@ -295,6 +295,13 @@ def comments(self):
             self._comments.append(Comment(user=user, **com))
         yield self._comments
 
+    def _progress(self, progress_type, specials=False):
```

Comment:
could you also add the `hidden` and `count_specials` parameters to this and the `collection_progress` and `watched_progress` methods?

Reply:
I thought to add things on demand, but ok!

Reply:
ee435c5, af1b70d